source/Octopus.Tentacle.Client/Capabilities/CapabilitiesResponseV2ExtensionMethods.cs

```diff
@@ -13,8 +13,7 @@ public static bool HasScriptServiceV2(this CapabilitiesResponseV2 capabilities)
                 return false;
             }
 
-            return capabilities.SupportedCapabilities.Contains($"{nameof(IScriptServiceV2)}Alpha") ||
-                capabilities.SupportedCapabilities.Contains($"{nameof(IScriptServiceV2)}");
+            return capabilities.SupportedCapabilities.Contains($"{nameof(IScriptServiceV2)}");
```

Comment:
Nit: Why `.Contains($"{nameof(IScriptServiceV2)}")` instead of simply `.Contains(nameof(IScriptServiceV2))`?

Reply:
Good spot, just a find Alpha and replace with nothing bug ;)